manifest.yaml

```diff
@@ -1,6 +1,7 @@
 name: github.com/getoutreach/stencil-base
 ###Block(keys)
-modules: []
+modules:
+  - name: github.com/getoutreach/devbase
```

Comment:
If we're going to add dependencies to this module, we should probably add `stencil-circleci` too. Right now it's a manual step to migrate changes here.

Reply:
Hmmm I don't quite follow 😓

Reply:
Never mind my mistake. I thought this was the `services.yaml` file